static/bidder-info/readpeak.yaml

```diff
@@ -0,0 +1,15 @@
+endpoint: https://dsp.readpeak.com/header/prebid
+geoscope:
+  - EEA
+maintainer:
+  email: devteam@readpeak.com
```

Comment:
@readpeak-user,  Email is being sent from prebid team to verify `devteam@readpeak.com` address. Requesting to respond back on email thread.

Reply:
Replied

Reply:
received reply from `devteam@readpeak.com`